### PR TITLE
Add game-cheat Object

### PR DESCRIPTION
**Pull request by enes-usta**
See https://github.com/RudigerMorinDocter/MISP-GameCheats

M2 SSI - Threat Intelligence project

### CODE REVIEW

File: objects/game-cheat/definition.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    },
+    "creator": {
+      "description": "Individual and/or Group and/or Organization that created the cheat.",
+      "misp-attribute": "threat-actor",
+      "ui-priority": 1
+    },
+    "ig-cheat-behaviour": {
+      "description": "Describe the in-game behaviour of the cheat (e.g. You selected 'Aim Bot', here you can add details like 'Activate by pressing F7, Deactivate by pressing F8. Not detected be Easy Anti-Cheat.'",
+      "disable_correlation": true,
+      "misp-attribute": "text",
```
Comment (iglocska) [inline objects/game-cheat/definition.json:64]: I'd pick "comment" as the attribute type here. This is a little bit of a nit-picky detail, but generally comments are best to use when the text is not meant to be correlated or used as an indicator ever but rather acts as a textual description of ongoing activities.

**Comment by iglocska** [comment]
Lovely stuff, thanks a lot! :)

File: objects/game-cheat/definition.json [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        "Fly",
+        "Force field",
+        "Ghosting",
+        "God mode",
+        "Invicibility",
+        "Macros",
+        "RapidFire",
+        "Scripting",
+        "Wallhack",
+        "Win trading",
```
[the remark this answers is not in the document]
Comment (enes-usta) [reply objects/game-cheat/definition.json]: As you suggested, I kept technical cheats and added some.

File: objects/game-cheat/definition.json (hunk 1 above)
Comment (enes-usta) [reply objects/game-cheat/definition.json]: I didn't know and hadn't thought about it. Done, as you said.

File: objects/game-cheat/definition.json [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      "multiple": true,
+      "ui-priority": 2
+    },
+    "pricing": {
+      "description": "Cheatware price, 0 if free.",
+      "disable_correlation": true,
+      "misp-attribute": "text",
+      "ui-priority": 6
+    },
+    "webpage": {
```
[the remark this answers is not in the document]
Comment (enes-usta) [reply objects/game-cheat/definition.json:102]: You're right.

**Comment by adulau** [comment]
Thanks a lot for the contribution.

**Pull request merged**